cmd/otel-allocator/README.md

```diff
@@ -98,7 +98,7 @@ The easiest way to do this is to grab a copy of the individual [`PodMonitor`](ht
 
 # Usage
 
-The `spec.targetAllocator:` controls the TargetAllocator general properties. Full API spec can be found here: [api.md#opentelemetrycollectorspectargetallocator](../../docs/api.md#opentelemetrycollectorspectargetallocator)
+The `spec.targetAllocator:` controls the TargetAllocator general properties. Full API spec can be found here: [api/opentelemetrycollectors.md](../../docs/api/opentelemetrycollectors.md)
```

Comment:
the link now is incorrect, but the correct one should contain the reference:

```
https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api/opentelemetrycollectors.md#opentelemetrycollectorspectargetallocator
```

Reply:
Okay, I fixed it with what you told me
- ec86bf4980777a5432ab029b0281b0c0bb3e090f